distmv/studentst.go

```diff
@@ -0,0 +1,347 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distmv
+
+import (
+	"math"
+	"math/rand"
+	"sort"
+	"sync"
+
+	"golang.org/x/tools/container/intsets"
+
+	"github.com/gonum/floats"
+	"github.com/gonum/matrix/mat64"
+	"github.com/gonum/stat/distuv"
+)
+
+// StudentsT is a multi-variate Student's T distribution. It is a distribution over
```

Comment:
Although I actually like the term "multi-variate" as it emphasises its meaning --- multiple variables --- it is not generally hyphenated, normally it is written "multivariate".